pallets/settlement/src/benchmarking.rs

```diff
@@ -97,6 +97,7 @@ fn setup_legs<T>(
     o: u32,
     pause_compliance: bool,
     pause_restrictions: bool,
+    asset_mediators: u8,
```

Comment:
I think this one should always be the maximum number of asset mediators.  Since the settlement benchmarks/weights don't have a parameter for it.

Reply:
Or at least for all of the `add_instruction***` extrinsics and the execute extrinsics.